services/order.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime
+from db.models import Order, User, Ticket
+from django.db.models.query import QuerySet
+from django.db import transaction
+from services.movie_session import get_movie_session_by_id
+
+
+@transaction.atomic
+def create_order(tickets: list[dict], username: str, date: str = None) -> None:
+    user = User.objects.get(username=username)
+    order = Order.objects.create(user=user)
+
+    if date:
+        date = datetime.strptime(date, "%Y-%m-%d %H:%M")
```

Comment:
Consider adding error handling for `datetime.strptime` to manage cases where the date format might be incorrect, which could raise a `ValueError`.